src/queries.js

```diff
@@ -18,7 +18,7 @@ export const makeTriggersQuery = {
   definition: () => {
     return Q('io.cozy.triggers')
       .partialIndex({
-        worker: 'konnector'
+        worker: { $in: ['client', 'konnector'] } // client is for CLISK
```

Comment:
Sadly, as you explain in the description of this commit, this request does not fetch jobs. And these jobs are needed to compute the `current_state` attribute of the trigger which is needed to know if the trigger has any error.

I must admit I have some difficulties to follow the flow of the front code to know where and how it is used but if I try your branch in my local cozy, I don't have any error status displayed on konnector icons in home application.

Reply:
*sigh* looks like I missed this case. I guess we could query the triggers with job info at the moment we click on the konnector "group" as it is at his moment that we display the error icon, right?

Reply:
See [48e75e1](https://github.com/cozy/cozy-home/pull/2259/commits/48e75e10970ce599d347314b92a7017e24f5298e)